pages/[[server]]/list/[list]/index/accounts.vue

```diff
@@ -1,16 +1,153 @@
 <script setup lang="ts">
+import AccountSearchResult from '~/components/list/AccountSearchResult.vue'
+
 definePageMeta({
   name: 'list-accounts',
 })
 
+const inputRef = ref<HTMLInputElement>()
+defineExpose({
+  inputRef,
+})
+
 const params = useRoute().params
 const listId = computed(() => params.list as string)
 
-const paginator = useMastoClient().v1.lists.$select(listId.value).accounts.list()
+const $mastoList = useMastoClient().v1.lists.$select(listId.value).accounts
+const paginator = $mastoList.list()
+const accountsInList = ref((await useMastoClient().v1.lists.$select(listId.value).accounts.list({ limit: 0 })).map(account => account.id))
```

Comment:
There is a bug in masto.js -- setting the `limit` to 0 should return all accounts but it doesn't work atm. Opened a ticket: https://github.com/neet/masto.js/issues/1282

Reply:
I made this limit to 1000 while the masto.js ticket is still open :)